src/Umbraco.Web/Editors/MediaController.cs

```diff
@@ -278,6 +281,31 @@ public IEnumerable<MediaItemDisplay> GetByIds([FromUri]int[] ids)
         }
 
         /// <summary>
+        /// Searches media and returns a paged result
+        /// </summary>
+        /// <param name="query"></param>
+        /// <param name="pageNumber"></param>
+        /// <param name="pageSize"></param>
+        /// <param name="searchFrom"></param>
+        /// <returns></returns>
+        [HttpGet]
+        [FilterAllowedOutgoingMedia(typeof(IEnumerable<ContentItemBasic<ContentPropertyBasic, IMedia>>), "Items")]

```

Comment:
@emilwangaa  why do you need a list of `ContentItemBasic<ContentPropertyBasic, IMedia>` ? what is in there that isn't in EntityBasic that you require? When did the Content Picker start using Examine as the search?